style.css

```diff
@@ -2,229 +2,780 @@
   /* stylelint-disable length-zero-no-unit */
   margin: 0px;
 
-  --font-family: 'poppins', sans-serif;
+  @import url('https://fonts.googleapis.com/css2?family=Poppins&display=swap');
+  @import url('https://fonts.googleapis.com/css2?family=Crete+Round&display=swap');
+  @import url('https://fonts.googleapis.com/css2?family=Inter:wght@300&display=swap');
+
+  text-decoration: none;
+  list-style-type: none;
+  box-sizing: border-box;
 }
 
 body {
-  margin: 0px;
+  width: auto;
+  max-width: 100%;
+  height: auto;
+  overflow-x: hidden;
 }
 
-.section1 {
-  margin: 0px;
+/* start to homepage */
+.homepage-container {
+  display: grid;
+  grid-template-columns: 1349px;
+  justify-self: center;
+  height: 900px;
+  min-width: 1366px;
+  max-width: 100%;
+  background-color: #1c1a19;
+}
+
+.header {
   display: flex;
+  background-color: #3c3a39;
+  width: 1349px;
```

Comment:
- [ ] I have noticed that you are using a fixed width on the header. Due to this, the header can not be responsive and therefore it is causing a horizontal scroll. The traditional horizontal scrollbar burdens the user by requiring constant attention and greater physical effort to maintain the dragging 😞. Kindly fix this as it will be the step to solve the horizontal scroll effect in your portfolio ☺️.